yt_dlp/extractor/mxplayer.py

```diff
@@ -110,6 +110,11 @@ def _real_extract(self, url):
                 for frmt in dash_formats:
                     frmt['quality'] = get_quality(quality)
                 formats.extend(dash_formats)
+                dash_formats_h265 = self._extract_mpd_formats(
+                    format_url.replace('h264_high', 'h265_main'), video_id, mpd_id='dash-%s' % quality, headers={'Referer': url}, fatal=False)
```

Comment:
Does url always have `h364_high` in it? Can't it be `h264_main` as well? In that case, this is downloading the same manifest twice

Reply:
From my testing so far, it's always `h264_high`.

This is a temp fix anyways, ideally we'd want to get the proper url from the api.